zrml/parimutuel/src/weights.rs

```diff
@@ -1,30 +1,14 @@
-// Copyright 2023 Forecasting Technologies LTD.
-//
-// This file is part of Zeitgeist.
-//
-// Zeitgeist is free software: you can redistribute it and/or modify it
-// under the terms of the GNU General Public License as published by the
-// Free Software Foundation, either version 3 of the License, or (at
-// your option) any later version.
-//
-// Zeitgeist is distributed in the hope that it will be useful, but
-// WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-// General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
 
 //! Autogenerated weights for zrml_parimutuel
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: `2023-10-13`, STEPS: `10`, REPEAT: `1000`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: `2023-10-26`, STEPS: `10`, REPEAT: `1000`, LOW RANGE: `[]`, HIGH RANGE: `[]`
 //! WORST CASE MAP SIZE: `1000000`
-//! HOSTNAME: `Chralt-3.local`, CPU: `<UNKNOWN>`
+//! HOSTNAME: `zeitgeist-benchmark`, CPU: `AMD EPYC 7601 32-Core Processor`
```

Comment:
Is this our reference machine?

Reply:
Yes

Reply:
Mostly single core performance is relevant though.